### PR TITLE
Sets the minimun lenght and maximun at config

### PR DESCRIPTION
**Pull request by vankk**
As requested in the previous PR. Let me know if the AAC has a style code to follow.

### CODE REVIEW

File: config.php [hunk 1]
```diff
@@ -139,6 +139,11 @@
 	// won't be displayed if there is only one item (rookgaard for example)
 	'character_towns' => array(1),
 
+	// characters lenght
+	// This is the mininum and the maxinum length that a player can create a character. It is highly recommend the maxinum lenght be 21.
```
Comment (slawkens) [inline config.php]: Its "minimum", and "maximum"! :P

File: config.php [hunk 2]
```diff
@@ -139,6 +139,11 @@
...
+	'min_length' => 4,
```
Comment (slawkens) [inline config.php]: I would change it to "character_name_min_length".

File: config.php [hunk 3]
```diff
@@ -139,6 +139,11 @@
...
+	'maxinum_length' => 21
```
Comment (slawkens) [inline config.php]: I would change it to "character_name_max_length", so it's more clear what it does.

**Comment by slawkens** [comment]
I would use PSR-2: https://www.php-fig.org/psr/psr-2/

It's not used everywhere in this project yet, but I think we will follow this coding style in future.

Fix what I mentioned in comments, and I will merge this.

Thanks again!

**Comment by vankk** [comment]
I updated the codes as you asked, and in futures PR, I'll use this coding style. Also, there is a Trello with the TODO things?

**Comment by slawkens** [comment]
Thanks again!

No, we don't have Trello board. We use plain TODO file, located in the main directory in this repository.

**Pull request merged**